net-dialup/ppp/files/ppp-2.5.0-fix-linking-with-lld.patch

```diff
@@ -0,0 +1,50 @@
+Bug: https://bugs.gentoo.org/905442
+so we basically just swapped the order
+1. check pkg-config first, not manual -L /usr/lib
+2. if pkg-config succeeded, do not bother with -L /usr/lib
+(don't overwrite the result)
```

Comment:
Please update this patch with the more verbose commit message you included in the upstream PR.

Also include a link to the upstream PR.

Reply:
Does it look good now?